bin/plugin/restricted/selfDelPersonalAccess

```diff
@@ -14,13 +14,16 @@ my $remainingOptions = OVH::Bastion::Plugin::begin(
     header             => "removing personal access to a server from an account",
     userAllowWildcards => 1,
     options            => {
-        "protocol=s" => \my $protocol,
+        "protocol=s"   => \my $protocol,
+        "proxy-host=s" => \my $proxyHost,
+        "proxy-port=s" => \my $proxyPort,
         # undocumented/compatibility:
-        "user-any" => \my $userAny,
-        "port-any" => \my $portAny,
-        "scpup"    => \my $scpUp,
-        "scpdown"  => \my $scpDown,
-        "sftp"     => \my $sftp,
+        "user-any"     => \my $userAny,
+        "port-any"     => \my $portAny,
+        "scpup"        => \my $scpUp,
+        "scpdown"      => \my $scpDown,
+        "sftp"         => \my $sftp,
+        "proxy-user=s" => \my $proxyUser,
```

Comment:
Is the positioning of this option intentional, i.e. as an "undocumented/compatibility" option, instead of being just below the 2 other ones above?